File: schemas/full_detail_info_schema.py
from typing import List, Optional

from pydantic import BaseModel

from schemas.article_attributes_schema import ArticleAttributesSchema
from schemas.article_ean_schema import ArticleEANSchema
from schemas.articles_schema import ArticlesSchema
from schemas.et_producer_schema import EtProducerSchema
from schemas.suppliers_schema import SuppliersSchema


class FullDetailInfoSchema(BaseModel):
    normalized_article: str
    article_ean : Optional[ArticleEANSchema]
    article_schema : Optional[ArticlesSchema]
    detail_attribute: List[ArticleAttributesSchema]
    img_urls: List[str]
    supplier_from_jc: Optional[EtProducerSchema]
    supplier_from_td: Optional[SuppliersSchema]

File: OzonDomains/TecDocModels/FullDetailInfoSchema.cs
using Servcies.ApiServcies.TecDocApi.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OzonDomains.TecDocModels
{
    public class FullDetailInfoSchema
    {
        public string? NormalizedArticle { get; set; }
        public List<ArticleAttributesSchema> DetailAttribute { get; set; }
        public List<string> ImgUrls { get; set; }
        public EtProducerSchema? SupplierFromJc { get; set; }
        public SuppliersSchema? SupplierFromTd { get; set; }
    }
}
